src/components/operations/operation-details/ko/runtime/operation-details.html

```diff
@@ -338,14 +338,14 @@ <h2>GraphQL endpoint</h2>
 <!-- ko if: $component.consoleIsOpen -->
 <!-- ko ifnot: apiType() === 'graphql' -->
 <div class="detachable-right flex-grow animation-fade-in">
-    <operation-console class="test flex flex-column gql-console-height" data-bind="dialog: {}"
+    <operation-console class="flex flex-column h-100" data-bind="dialog: {}"
         params="{ api: api, operation: operation, hostnames: hostnames, authorizationServer: associatedAuthServer, useCorsProxy: useCorsProxy }">
     </operation-console>
 </div>
 <!-- /ko -->
 <!-- ko if: apiType() === 'graphql' -->
 <div class="detachable-right graphql-console-width flex-grow animation-fade-in">
-    <graphql-console class="gql-console-height flex flex-column" data-bind="dialog: {}"
+    <graphql-console class="flex flex-column h-100 gql-console-bg" data-bind="dialog: {}"
```

Comment:
We have a style defined for operation-console that has the background and height:
![image](https://user-images.githubusercontent.com/92857141/173028470-c29d3c02-1329-4bc3-90be-7a5747cb9e87.png)

Since we want the same background and height for `operation-console` and  `graphql-console`, we could create a class that has these properties and use it for both of them(and remove the operation-console style). 

Or, if you think it's better to have separate styles for them, maybe we can create a similar style for the `graphql-console`?

What do you think?

Reply:
Both consoles seem to have some particular styles. But I think we could do something like this to define what would be common and what would be independent for both. Thanks.
![image](https://user-images.githubusercontent.com/88548172/173134290-228b3790-9690-46ae-895e-da1e84144786.png)
![image](https://user-images.githubusercontent.com/88548172/173134689-d41fb7ef-0283-442d-ad49-807a89921446.png)